### PR TITLE
Nominate Mads Hartmann for LSP-WG

### PR DESCRIPTION
**Pull request by mads-hartmann**

### CODE REVIEW

**Comment by propensive** [comment]
@laughedelic @dragos Would either of you like to nominate and second @mads-hartmann? This should just be a formality. If you just comment here, I'll update the PR when I merge it.

File: nominations/mads-hartmann.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- Wrote one of the [more popular][scala.tmbundle] Scala syntax highlighting
+  definitions way back when I was using TextMate. I believe most editors
+  indirectly use this definition for highlighting (don't quote me on that). I
+  don't think it is directly relevant to the work we'll be doing in the LSP-WG
+  but it has convinced me that you can only do such much with regular
+  expressions before everything becomes one big mess 😉
+
+## Affiliations
+
+None
```
Comment (propensive) [inline nominations/mads-hartmann.md]: Just to confirm, even if you're not representing any organization, any affiliations (e.g. through employment) should be listed here, just for transparency.

**Comment by laughedelic** [comment]
@propensive unfortunately, I have never worked together with @mads-hartmann and don't know him personally, so I don't know if it's a valid endorsement, but I've seen him collaborating on the ENSIME integration for Atom: https://github.com/ensime/ensime-atom and I think it's a very relevant experience for the topic.

File: nominations/mads-hartmann.md (hunk 1 above)
Comment (mads-hartmann) [reply nominations/mads-hartmann.md]: Oh, I'll add my employer then :)

**Comment by fommil** [comment]
I'm happy to give a nomination or second, as appropriate.

Although, unless you're actually representing your employer, you should probably make Affiliations read as "independent".

**Comment by mads-hartmann** [comment]
Thanks @fommil, I appreciate it. I'm not representing my employer in this case. If I get elected I will be participating in this working group on my own time.

**Comment by dragos** [comment]
I'm happy to nominate and work with Mads, should we both get elected.

**Review by propensive — APPROVED** [review]

**Comment by mads-hartmann** [comment]
@dragos @fommil @propensive Thanks!

**Pull request merged**